controllers/usersController.js

```diff
@@ -0,0 +1,72 @@
+const db = require("../db/models");
+const User = db.users;
+
+const Op = db.Sequelize.Op;
+
+function createNewUser(req, res, next) {
+  const { email, password, role, firstname, lastname, gender, dob } = req.body;
+  User.create({
+    email,
+    password,
+    role,
+    firstname,
+    lastname,
+    gender,
+    dob,
+  })
+    .then((data) => {
+      res.status(201).send({ data });
+    })
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+}
+function verifyUser(req, res, next) {
+  const { token } = req.params;
+  const { email } = req.body;
+
+  User.findByPk(email)
```

Comment:
Don't make email your primary key. Use an autoincrement value or UUID. Email should just be unique